workbench/tests/renderer/investtab.test.js

```diff
@@ -266,39 +266,6 @@ describe('Sidebar Buttons', () => {
     expect(archiveDatastack).toHaveBeenCalledTimes(1);
   });
 
-  test('Multiple Save Clicks: each triggers a unique alert', async () => {

```

Comment:
This test seemed extraneous. I think the original intent was to test behavior of someone rapidly clicking all the save buttons. But that's much harder to do now that they are nested inside a `Save as...` dialog. We have assertions for the alerts in other tests.